dspace/config/spring/rest/event-service-listeners.xml

```diff
@@ -21,4 +21,9 @@
         <property name="eventService" ref="org.dspace.services.EventService"/>
     </bean>
 
+    <!--  Irus statistics tracking  -->
+<!--    <bean class="org.dspace.statistics.export.ExportUsageEventListener">-->
+<!--        <property name="eventService" ref="org.dspace.services.EventService"/>-->
+<!--    </bean>-->
```

Comment:
Configuring this feature should not require users to edit Spring bean definitions.  (Spring bean level configurations should really be reserved for advanced configurations which most users need not touch.)

I'd recommend this be enabled/disabled via a configuration in `local.cfg` in the same (or similar) way to Google Analytics (and the `GoogleRecorderEventListener` in this same Spring XML config).

Reply:
Also, this property looks to me like it should be marked @Inject or @Autowired instead of configured, since it likely will never be REconfigured.  I would expect that we'd have a different implementation of EventService, if this had to change, because it would be changing for all dependents.  But that's a guess....

@tdonohue we need to work out and write down guidance on how we all expect to see DI used in DSpace and how to classify configuration (including bean wiring).  Not here of course, but somewhere prominent.

Reply:
@mwoodiupui : Agreed on writing this down better...I simply haven't had time (and I welcome collaboration on this if you are interested).  But, yes, `@Autowired` could make sense here, and I prefer that for things not needing to be touched.  Spring bean XML is ok for things that might need touching, but only for *advanced configs* (though quite honestly, I prefer `@Autowired` where possible to do so, and I've tried to keep Spring XML entirely out of the new `dspace-server-webapp`).  Finally, `*.cfg` configs is for things that require extra configuration or for enabling/disabling features (even enabling/disabling OAI-PMH, SWORD,etc is now in `*.cfg` files as of DSpace 7).

Reply:
Thanks for the feedback! The integration can now be enabled with a config property stats.tracker.enabled in stats.cfg.